src/models/gamma_poisson_model.py

```diff
@@ -131,6 +131,7 @@ def __init__(
             the inventory is at least twice the number of impressions that were
             observed in the campaign.
         """
+        super().__init__(data, max_reach)
```

Comment:
This line is Ok but not useful?

`self._data` seems not used anywhere.   `self._max_reach` has been specified in line #144 below.

Reply:
See `src/driver/experimental_trial.py`, line 300.